packages/react-router/typedoc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "entryPoints": ["./index.ts"],
+  "entryPoints": ["./index.ts", "./dom-export.ts"],
```

Comment:
We've specifically avoided doing this so far because it causes an extra level of nesting that we don't want the UI - it now look like these APIs should be imported from `react-router/index` and `react-router/dom-export` - both of which are incorrect...

<img width="576" height="720" alt="Screenshot 2025-07-21 at 5 39 15 PM" src="https://github.com/user-attachments/assets/2a317692-eae8-44fa-a5bb-883cdbd99c55" />


I don't yet know the right solution but I think it's ok to omit the dom exports from typedoc for now and just document them in the MD files.